### ISSUE TITLE
Eventi in streaming

### ISSUE THREAD

**Issue by AlessandroMinoccheri** [DevRomagna/eventi#57]
Ciao a tutti,
cosa ne pensate di mettere gli eventi in streaming o registrarli per poi metterli su qualche canale a disposizione di tutti?
Ieri sera ne parlavo con @nicolaiarocci e l'idea è piaciuta ma ha anche dei contro.
Un vantaggio è per chi è lontano e non riesce a venire nel luogo prestabilito può vedersi l'evento comodamente da dove vuole, possiamo aumentare così la visibilità della community.
Un contro potrebbe essere il fatto che le persone non vengano più ai meetup vista la presenza dei video o dello streaming.

Cosa ne pensate? Idee?

**Comment by nicolaiarocci** [comment]
Come dicevo ieri sera, un rischio magari non alto è che cali la frequenza. In Romagna dobbiamo sobbarcarci spesso qualche km extra per partecipare... se possiamo evitarcelo, la tentazione di non andare potrebbe essere forte laddove invece l'incontro di persona, fare nuove conoscenze, eccetera è importante (se questo ultimo aspetto dovremmo lavorare un po). 

In ogni caso più che lo streaming penserei ad una eventuale registrazione da postare su YouTube in seguito, anche per ridurre i problemi tecnici (wifi scarso, ecc.). Inoltre, ovviamente, occorre avere sempre una camera con treppiede a disposizione. In DevMarche alla fine se la sono comprata :-)

**Comment by nicolaiarocci** [comment]
Sentiamo cosa ne pensa anche @micheleorselli (e ovviamente tutti gli altri)

**Comment by bago** [comment]
Faccio il rompica***

Il valore più grande di questi incontri, secondo me, è INCONTRARSI: il rischio che mettendo streaming/registrazioni cali l'affluenza dal vivo mette a rischio l'intero progetto e non da grossi vantaggi al progetto stesso a meno che non lo si pensi come progetto di Formazione 1:n .. ma allora tanto vale fare dei webinar.

Lo streaming o i video postregistrazione servono per "accorciare le distanze" e a quel punto perchè limitarsi a DevRomagna? Diventerebbero webinar per l'intera comunità di Dev italiani.. perchè dovrei vedere l'incontro di DevRomagna invece di quello di DevSicilia? E perchè allora ogni Dev "locale" dovrebbe fare gli stessi contenuti su youtube?

La presenza della telecamera potrebbe accentuare le timidezze sia di relatori che di pubblico partecipante.

Poi succedere 2-3 volte a sera (se va bene) il "mi scrivono che dallo streaming non si vede bene" "mi scrivono che non si sente" "è finita la memoria della videocamera"...

**Comment by sensorario** [comment]
Dopo aver letto i vostri commenti, ... trovo piu' interessante la pubblicazione di un video post evento. Ci eviteremmo la gestione di chi ci ascolta e ci guarda (wifi o altro), ma arricchiremmo comunque le risorse disponibili per la/le community.

**Comment by micheleorselli** [comment]
La penso come @bago: la cosa bella di questi incontri è il fatto che ci si vede fisicamente, ma non credo che pubblicare video farebbe diminuire il numero di partecipanti, al contrario la vedo come una forma di promozione. 
Gestirei però la cosa in maniera "lean": c'è qualcuno che se ne vuole occupare? Se la risposta è no per me i video possono aspettare, anche considerato che siamo un gruppo molto giovane ed abbiamo fatto pochi incontri. 

Sullo streaming live, lo vedo ad un livello di difficoltà ulteriore rispetto a produrre un video e pubblicarlo non in realtime: bello, ma ci penserei quando siamo più rodati

Gli altri che ne pensano?

**Comment by Fuminshou** [comment]
Concordo anche io con @bago, il valore aggiunto di questi incontri è la possibilità di vedersi faccia a faccia, fare 4 chiacchiere, entrare in contatto con altre persone con le quali instaurare un rapporto o (perchè no) una collaborazione lavorativa.

Ridurre tutto a un video ci porterebbe a ridurre ulteriormente i rapporti interpersonali, e temo anche che in poco tempo non sarebbero seguiti nemmeno i video, proprio perchè verrebbe a mancare la componente umana. Chi me lo fa fare di vedere un video amatoriale sull'argomento X se online ci sono già dei webinar?

**Comment by valentinavirga** [comment]
Concordo. 
La cosa bella e' creare un contatto e far chiacchere. A scoprire cose nuove da soli piu' o meno siam capaci. La rete e' piena di tutorial, webinar e cose cosi'. E concordo anche con la timidezza del relatore. 
Il video post incontro e' una bella idea. Ma per il momento e' carino anche che venga pubblicato altro materiale come le slides.

**Comment by ghost** [comment]
Ciao ragazzi, scusate se commento un argomento un pò vecchio, ma ha catturato la mia attenzione.
Capisco ragazzi che il contatto e far due chiacchiere è la "bellezza" di questi incontri.
Nel mio caso però, mi piacerebbe molto seguire i corsi ma avendo pochissimo tempo faccio davvero fatica a partecipare fisicamente. Io pur di seguirvi sarei favorevole ad una differita su Youtube.
